CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## unreleased
 
+* Updated all dependencies to the latest including eslint-plugin-jest 27.
 * Updated `eslint-plugin-react` version to `^7.32.2`.
```

Comment:
The change log of eslint-plugin-react is included in the module update written above, but it is the log that the restoration of the module version pinning in cli 5.1.1, so I think it needs to be kept.
Could I keep this log? or should remove it?

Reply:
You don't need to modify another PR's changelog. Please leave as -is.